src/components/toolbox/onboarding/onboarding.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './onboarding.css';
+
+class Onboarding extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      step: 0,
+    };
+  }
+
+  render() {
+    const { slides } = this.props;
+    const { step } = this.state;
+
+    return slides.length ? (
+      <div className={styles.onboarding}>
+        <span className={styles.closeBtn} />
+        {slides.map(({ className = '', ...slide }, index) => (
+          <section key={`slides-${index}`}
+            style={{ backgroundImage: `url(${slide.illustration})` }}
+            className={`${className} ${index === step ? 'active' : ''}`}>
+            <h1 className={styles.title}>{slide.title}</h1>
+            {typeof slide.content === 'string' &&
```

Comment:
https://github.com/LiskHQ/lisk-hub/blame/development/docs/REACT_GUIDE.md#L43

Reply:
Oh, I know, this is just a placeholder, since it may be possible to have different types of content, I'll create a function to format in the correct way, but for now, I was just testing with strings.

Reply:
Ah, got it.